Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:16.04
+MAINTAINER Devin Ekins <devops@keen.io>
+
+# Setup the dependencies
+RUN apt-get update -y
+RUN apt-get install -y apt-utils
```

Comment:
These should all be one run command.  That way you don't generate unnecessary layers. https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#/sort-multi-line-arguments

```
RUN apt-get update -y && \
 apt-get install -y apt-utils libtool make automake
```